packages/webdriver/src/utils.js

```diff
@@ -218,13 +218,18 @@ export function isChrome (caps) {
  */
 export function isMobile (caps) {
     return Boolean(
-        (typeof caps['appium-version'] !== 'undefined') ||
+        (typeof caps['appium-version'] !== 'undefined') || (typeof caps['appiumVersion'] !== 'undefined') ||
         (typeof caps['device-type'] !== 'undefined') || (typeof caps['deviceType'] !== 'undefined') ||
         (typeof caps['device-orientation'] !== 'undefined') || (typeof caps['deviceOrientation'] !== 'undefined') ||
         (typeof caps.deviceName !== 'undefined') ||
+        // experitest returns actual device capabilities in the form "device.name" | "device.model" etc.
+        (typeof caps['device'] !== 'undefined' && (
+            (typeof caps['device.name'] !== 'undefined') ||
```

Comment:
I see lots of common when checking properties like deviceName, device.name, device-name.
What do you think on adding function that accepts caps, device, name and verifies all 3 combinations.